backend/app/src/main/kotlin/io/tolgee/api/v2/controllers/AdministrationController.kt

```diff
@@ -49,6 +51,8 @@ class AdministrationController(
 
   @GetMapping(value = ["/organizations"])
   @Operation(summary = "Get all server organizations")
+  @NeedsSuperJwtToken
+  @DenyPatAccess
```

Comment:
I would keep it separate. I am not sure if there won't be a reason to have @NeedsSuperJwtToken without @DenyPatAccess in the future.

Reply:
In that case, there are many places where a `@DenyPatAccess` is missing; I don't think out of all the endpoints currently marked with NeedsSJWT there's one that should be available with a PAT

Reply:
Anyway, this way it's clear what happens. So instead of creating new annotation for like `@NeedsSuperJwtTokenAndDenyPatAccess` I am going to keep both, which is clear.